Enterprise-CMCS/macpro-mdct-mcr
# Add ETL for adding PCCM as "no" to mcpar metadata on reports created prior to PCCM option

### Description


In MCR val you can see this bug where reports created before PCCM have no value for that radio, it is disabled, and you cannot edit a report because it is required.

https://github.com/Enterprise-CMCS/macpro-mdct-mcr/assets/57802560/6a8994b5-51e6-4cce-8234-74496c8994d8

This ETL adds the metadata value for PCCM as 'no' to all reports created before PCCM, which accurately reflects what those reports are.


### Related ticket(s)


CMDCT-3049

---
### How to test

Setup
- Run MCR locally
- Login as a state user and create two reports

Create "report without PCCM in metadata"
- In a new terminal tab run dynamodb-admin with `DYNAMO_ENDPOINT=http://localhost:8000 dynamodb-admin`
  - Open localhost:8001
  - Go the local-mcpar-reports table and select one of the items
  - Remove the `programIsPCCM` data and save that item
- Refresh MCR in your browser. Verify that when you click the edit icon one shows up without a radio for pccm selected, that radio is disabled, and it will not let you save without selecting an option

Run the script to fix that report
- In a new terminal tab navigate to services/app-api in the MCR directory
- Run `serverless invoke local --function addPccmToPastMcpar`

Verify
- Refresh the dynamodb-admin browser tab and verify the item that you removed pccm from now has it again set to 'no'
- Refresh the MCR browser tab and verify that both reports can be edited and saved, and that pccm is disabled and shows the matching choice

---
### Author checklist


---
